src/languages/sqf.js

```diff
@@ -1858,7 +1840,7 @@ export default function(hljs) {
     'setCurrentTask',
     'setCurrentWaypoint',
     'setCustomAimCoef',
-    'setCustomMissionData',
+    'SetCustomMissionData',
```

Comment:
These changes (at a glance) seem almost random.  Are you carefully making them by hand or is there some script you are using to fetch the keyword list directly from Arma?

Reply:
And the whole grammar is flagged `case_insensitive`, so is there truly a correct casing here at all - that keeps changing from version to version?

Reply:
We use this highlighter for the Arma 3 Discord where casing doesn't really matter.
The casing is set in the keyword description in the engine and are fetched via a script. The casing seems random and inconsistent because they are set like this in engine.

Reply:
Can we get a link to the script you are using added to the source in comments for the aid of future maintainers? (or if it's super short we could include the whole script in a comment)

Reply:
I wrote it myself

Reply:
~4 lines is pretty short.   You can attach it here so we can have a look at it.

Reply:
@joshgoebel It was a bit longer than what I thought
```sqf
/*
	# Author: Leopard20
	
	# Description:
	This script can be used to dump all commands to clipboard..
	Make sure you're using the Diag EXE to dump all of the commands.
	
	# How to use:
	Simply copy the array of KEYWORDS and LITERALS from this sqf.js file and paste it in the script.
	Execute the script from the debug console.
	All commands will be copied to clipboard
*/

_KEYWORDS = ['if'];                                                //Array of all KEYWORDS
_LITERALS = ['west'];                                              //Array of all LITERALS
_allCommands = createHashMap;
{
	_type = _x select [0,1];
	if (_type != "t") then {
		_command_lowerCase = ((_x select [2]) splitString " ")#(((["n", "u", "b"] find _type) - 1) max 0);
		_command_upperCase = supportInfo ("i:" + _command_lowerCase) # 0 # 2;
		_allCommands set [_command_lowerCase, _command_upperCase];
	};
} forEach supportInfo "";
_allCommands = _allCommands toArray false;
_allCommands sort true;                                            //sort by lower case
_allCommands = ((_allCommands apply {_x#1}) -_KEYWORDS)-_LITERALS; //remove KEYWORDS and LITERALS
copyToClipboard str (_allCommands select {_x regexMatch "\w+"});
```

Reply:
correct script command casing is supposed to be camelCase.
Sometimes in engine we do CamelCase for incomplete things, and convert it to correct one later.
Sometimes thats also forgotten